### PR TITLE
Headers insensitive

### LINKED ISSUE #62: Headers case-insensitivity
https://github.com/probot/adapter-aws-lambda-serverless/blob/d126582a12aa54776d34e8bfccdc8ce41f7df3b9/lambda-function.js#L11-L14

From [RFC 7230 > 3.2.  Header Fields](https://tools.ietf.org/html/rfc7230#section-3.2):

> Each header field consists of a case-insensitive field name followed by a colon (":"), optional leading whitespace, the field value, and optional trailing whitespace.

=> If you are testing (sample #61) your Lambda Probot with some tools which can "rewrite" headers (sample: [Ansible URI module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html)) like that (in Camel Case):

```
X-Github-Delivery: fake-ansible
X-Github-Event: test
```
*Note the 'h' in lowercase*

It doesn't work 😢. It would be nice if *adapter-aws-lambda-serverless* could be **headers case-insensitive**.

Error detail:
```
INFO	AggregateError: 
    Error: Event name not passed
        at Array.map (<anonymous>)
        at receiverHandle (/var/task/node_modules/@octokit/webhooks/dist-node/index.js:98:11)
        at verifyAndReceive (/var/task/node_modules/@octokit/webhooks/dist-node/index.js:307:29)
        at lambdaFunction (/var/task/node_modules/@probot/adapter-aws-lambda-serverless/lambda-function.js:12:24)
        at Runtime.handleOnce (/var/runtime/Runtime.js:66:25)
    at receiverHandle (/var/task/node_modules/@octokit/webhooks/dist-node/index.js:98:11)
    at verifyAndReceive (/var/task/node_modules/@octokit/webhooks/dist-node/index.js:307:29)
    at lambdaFunction (/var/task/node_modules/@probot/adapter-aws-lambda-serverless/lambda-function.js:12:24)
    at Runtime.handleOnce (/var/runtime/Runtime.js:66:25)
```
*NB: False line number, console.log added for debug*

### PR DESCRIPTION
**Pull request by axel3rd**
fixes #62

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -93,4 +94,78 @@ describe("@probot/adapter-aws-lambda-serverless", () => {
...
+    await fn(event, context);
+
+    expect(context).toStrictEqual({
+      callbackWaitsForEmptyEventLoop: false,
+    });
+
+    expect(mock.activeMocks()).toStrictEqual([]);
+  });
+
+  test("camelcase request headers", async () => {
```
Comment (gr2m) [inline test/index.test.js:135]: Maybe add a reference to the issue as it includes relevant background information about the casing of the headers in this test

```suggestion
  test("camelcase request headers (#62)", async () => {
```

**Review by gr2m — COMMENTED** [review]
tests look good so far 👍🏼

**Comment by axel3rd** [comment]
**NB**: `package-lock.json` not commited, I'm NOT on *https://registry.npmjs.org* endpoint => pollution with different URL (on `lowercase-keys` package)

File: lambda-function.js [hunk 2]
```diff
@@ -1,22 +1,23 @@
 module.exports = lambdaFunction;
 
+const lowercaseKeys = require("lowercase-keys");
+
 async function lambdaFunction(probot, event, context) {
   try {
     // Ends function immediately after callback
     context.callbackWaitsForEmptyEventLoop = false;
 
+    // lowercase all headers to respect headers insensitivity (RFC 7230 $3.2 'Header Fields', see issue #62)
```
Comment (gr2m) [inline lambda-function.js:10]: thanks for adding the comment & referencing the issue 👍🏼

**Review by gr2m — APPROVED** [review]
Looks great, thank you!

**Pull request merged**